### DISCUSSION TITLE
can’t share the same workspace in multi devices

### DISCUSSION THREAD

**Discussion by LYUYork** [anyproto/contributors#230] [Any-Sync]
I want to edit the workspace in my pc and my mac.

**Comment by fuksman** [comment]
Hello, you don't need to share it; just use the same recovery phrase to log in from different devices.

**Reply by LYUYork** [reply]
don't work
<img width="641" alt="image" src="https://github.com/user-attachments/assets/c7da1eb3-d986-476b-9d17-adda0c668ca4">

**Reply by fuksman** [reply]
Please ensure that you use the same recovery phrase on your PC and Mac and that the version of Anytype is the same between devices. The message from your screenshot appears for old account, which didn't migrate to a new version of underlying protocol last year. If you successfully use your account on one of the devices with a new version of Anytype, everything should have migrated already.
